### DISCUSSION TITLE
x-show.transition.origin.top.right isn't applied when leaving

### DISCUSSION THREAD

**Discussion by alexsanderson** [alpinejs/alpine#1410] [7. Bugs (archived)]
In the GIF below, see that the `origin.top.right` is applied on the entering phase, but not the leaving phase. However, adding a scale fixes this. Following the web inspector, I can see that the correct CSS properties are being applied, but the result isn't seen in the UI. 

![ezgif com-gif-maker](https://user-images.githubusercontent.com/25802369/96699432-2ad3c880-1386-11eb-8baf-ad1f6bbea682.gif)

Here's the code: [https://jsfiddle.net/6gvt5Lnj/3/](https://jsfiddle.net/6gvt5Lnj/3/)

I've slowed it down so it's easier to see, but it was noticeable at the default duration (200ms), hence the reason for investigating.

Note: This bug is intermittent (works around 1 in 5 times), so you might need to re-run the fiddle to see it.

As the classes are being applied, I wondered is this a timing issue or something else within Apline's domain, or is it a browser bug?

**Comment by molfar** [comment]
Confirm, have encouraged the same issue. Leave transition does not work as expected, origin values not respected.
